src/components/ProjectDisciplines.js

```diff
@@ -37,7 +44,8 @@ class ProjectDisciplines extends React.Component {
           title={label}
           tag={value}
           key={idx}
-          color={color} /> )
+          color={color}
+          setSelectedProjectTag={() => {this.props.setSelectedProjectTag(value)}} /> )
```

Comment:
Hmm, it's in the proptypes and it's being called in handleClick.  Is that what you were looking for or ?